containers/src/Kokkos_ScatterView.hpp

```diff
@@ -177,18 +177,19 @@ struct DefaultContribution<Kokkos::Cuda,
    etc. Note the addition of update(ValueType const& rhs) and reset()  so that
    all reducers can have common functions See ReduceDuplicates and
    ResetDuplicates ) */
-template <typename ValueType, int Op, int contribution>
+template <typename ValueType, int Op, typename DeviceType, int contribution>
```

Comment:
Why did you chose not to add it I last position?

Reply:
I put the template parameter in this position so that the parameter order would be consistent with the order in `ScatterView`